configs/11.0/packages/tcmalloc/sources

```diff
@@ -41,3 +41,20 @@ ATSRC_PACKAGE_TARS=""
 ATSRC_PACKAGE_MAKE_CHECK=""
 ATSRC_PACKAGE_DISTRIB=no
 ATSRC_PACKAGE_BUNDLE=toolchain_extra
+
+
+atsrc_get_patches ()
+{
+	# Fix ucontext issue.
+	# Remove this when gperftools PR #908 is merged.
+	at_get_patch \
+		https://github.com/tuliom/gperftools/commit/99089d65d11f.patch \
+		67e4d8d1c2876c25f94ffb257392325b || return ${?}
```

Comment:
One question. This patch was sent to places like patchwork. Why do you apply and make a patch from the commit on your personal github repo instead just point to patchwork link? I have no problem with that it's just a question.